packages/frameworks/frameworks.json

```diff
@@ -168,6 +168,37 @@
       }
     }
   },
+  {
+    "name": "Dojo",
+    "slug": "dojo",
+    "demo": "https://dojo.now-examples.now.sh",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/dojo.svg",
+    "tagline": "Dojo is a modern progressive, TypeScript first framework.",
+    "description": "A Dojo app, created with the Dojo CLI's cli-create-app command.",
+    "website": "https://dojo.io",
+    "detectors": {
+      "some": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@dojo\\/framework\":\\s*\".+?\"[^}]*}"
+        },
+        {
+          "path": ".dojorc"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`dojo build` or `build` from `package.json`"
+      },
+      "devCommand": {
+        "value": "dojo build -m dev -w -s"
```

Comment:
Is there a a PORT that can be passed here?

Reply:
Yes a port can be passed using the `-p` flag